server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('./webpack.config.js');
+
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: config.output.publicPath,
+  }),
+);
+
+// Serve the files on port 3000.
+app.listen(3000, () => {
+  // console.log('Example app listening on port 3000!\n');
```

Comment:
[OPTIONAL]
- Remove all the commented line of code as removing commented code makes your codebase cleaner.